Source/KISAddonConfig.cs

```diff
@@ -71,6 +71,9 @@ class KISPodInventoryLoader: LoadingSystem {
           AddPodInventories(avPart.partPrefab);
         }
       }
+      for (var i = 0; i < QueuedPodInventoryParts.Count; i++) {
```

Comment:
If I read the code correctly, removing condition `avPart.partPrefab.CrewCapacity > 0` at line 70 should have the same effect.